requirements.txt

```diff
@@ -4,3 +4,4 @@ matplotlib
 xarray
 omfit_eqdsk
 omfit_gapy
+pexpect
```

Comment:
@StanczakDominik I think that pexpect is needed by IPython, which isn't strictly necessary in Aurora, but is the way I personally tested the package `examples`. Thanks for adding it to the requirements!